### PR TITLE
Mutation functions to allow coercion

### PR DESCRIPTION
**Pull request by anderson-joyle**
Issue #1205.

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Interpreter/Functions/Extensions.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ internal static bool IsValid(this DateTime dateTime, TimeZoneInfo tzi)
         /// <param name="usePowerFxV1CompatibilityRules">Use PFx v1 compatibility rules if enabled (less
         /// permissive Accepts relationships).</param>
         /// <returns></returns>
-        internal static bool CheckAggregateNames(this DType argType, DType dataSourceType, TexlNode arg, IErrorContainer errors, bool supportsParamCoercion, bool usePowerFxV1CompatibilityRules)
+        internal static bool CheckAggregateNames(this DType argType, DType dataSourceType, TexlNode arg, IErrorContainer errors, bool supportsParamCoercion = false, bool usePowerFxV1CompatibilityRules = false)
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Interpreter/Functions/Extensions.cs:52]: >= false [](http://example.com/codeflow?start=156&length=7)

why make these default params?

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**